pinax/api/jsonapi.py

```diff
@@ -43,7 +46,7 @@ def from_validation_error(cls, exc, resource_class):
                 errs.append(err)
         return cls(errors=errs)
 
-    def __init__(self, data=None, errors=None, links=False, included=None, meta=None, linkage=False):
+    def __init__(self, data=None, errors=None, links=False, included=None, meta={}, linkage=False):
```

Comment:
Simple fix: check for `None` in `__init__` and if true then set `self.meta` to a dictionary created in the method scope.